Simplenote/src/main/res/layout/activity_notes.xml

```diff
@@ -36,15 +36,37 @@
 
     </LinearLayout>
 
-    <com.google.android.material.navigation.NavigationView
-        android:id="@+id/navigation_view"
-        android:background="?attr/drawerBackgroundColor"
-        android:layout_gravity="start"
-        android:layout_height="match_parent"
+    <LinearLayout
         android:layout_width="wrap_content"
-        app:insetForeground="@android:color/transparent"
-        app:itemShapeFillColor="?attr/drawerBackgroundSelector"
-        app:theme="@style/NavigationMenuItem">
-    </com.google.android.material.navigation.NavigationView>
+        android:layout_height="match_parent"
+        android:layout_gravity="start"
+        android:background="?attr/drawerBackgroundColor"
+        android:elevation="@dimen/navigation_elevation"
+        android:fitsSystemWindows="true"
+        android:orientation="vertical">
+
+        <com.google.android.material.navigation.NavigationView
+            android:id="@+id/navigation_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:elevation="0dp"
```

Comment:
```suggestion
            android:background="?attr/drawerBackgroundColor"
            app:elevation="0dp"
```

We'll want to keep the `android:background="?attr/drawerBackgroundColor"` attribute and value for the `NavigationView` view so that the background color matches the rest of the navigation drawer.  It's not noticeable with the **_Light_** theme since it uses the default background color, but it's apparent with the **_Dark_** theme.  It's more apparent with other styles too.  See the screenshots below for illustration.

![1249_navigation_tags_hint_portrait](https://user-images.githubusercontent.com/3827611/103015849-58f89480-44fe-11eb-8ac7-98a214b0269e.png)

Reply:
Nice catch 👍